libs/sdk/skyux-stylelint/README.md

```
@@ -1,3 +1,20 @@
-# skyux-stylelint (Experimental)
+# skyux-stylelint
 
-Stylelint rules for SKY UX projects.
+## Install
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)